packages/ui-top-nav-bar/src/TopNavBar/README.md

```diff
@@ -426,28 +436,33 @@ class PlaygroundExample extends React.Component {
 
             const checkboxSettings = checkboxes.length ? (
               <CheckboxGroup
-               description={label}
-               name={name}
-               key={name}
-               layout="stacked"
-               defaultValue={checkboxes.filter(setting => this.state[setting])}
+                description={label}
+                name={name}
+                key={name}
+                layout="stacked"
+                defaultValue={checkboxes.filter(setting => this.state[setting])}
               >
-               {checkboxes.map((setting) => (
-                 <Checkbox
-                   key={`${setting}Setting`}
-                   variant="toggle"
-                   value={setting}
-                   label={settingLabels[setting] || `"${setting}" prop`}
-                   checked={this.state[setting]}
-                   messages={settingDescriptions[setting]
-                     ? [{ text: settingDescriptions[setting], type: 'hint' }]
-                     : undefined
-                   }
-                   onChange={() => {
-                     this.setState({ [setting]: !this.state[setting] })
-                   }}
-                 />
-               ))}
+                {checkboxes.map((setting) => {
+                  if (setting === 'hasBrandSection') {
+                    console.log(this.state[setting])
```

Comment:
Did you leave this console.log here on purpose?

Reply:
no, will remove it